python/lsst/ts/Dome/dome_csc.py

```diff
@@ -387,47 +405,34 @@ async def do_setTemperature(self, data):
         """
         self.assert_enabled()
         await self.write_then_read_reply(
-            command="setTemperature", parameters={"temperature": data.temperature}
+            command="setTemperature", temperature=data.temperature
         )
 
-    async def config_llcs(self, data):
+    async def config_llcs(self, system, settings):
         """Config command not to be executed by SAL.
 
         This command will be used to send the values of one or more parameters to configure the lower level
         components.
 
         Parameters
         ----------
-        data : `dict`
-            A dictionary with arguments to the function call. It should contain keys for all lower level
-            components to be configured with values that are dicts with keys for all the parameters that
-            need to be configured. The structure is::
-
-                "AMCS":
-                    "jmax"
-                    "amax"
-                    "vmax"
-                "LWSCS":
-                    "jmax"
-                    "amax"
-                    "vmax"
+        system: `str`
+            The name of the lower level component to configure.
+        settings : `dict`
+            A dict containing key,value for all the parameters that need to be configured. The structure is::
 
-        It is assumed that configuration_parameters is presented as a dictionary of dictionaries with one
-        dictionary per lower level component. This means that we only need to check for unknown and too
-        large parameters and then send all to the lower level components. An example would be::
-
-            {"AMCS": {"amax": 5, "jmax": 4}, "LWSCS": {"vmax": 5, "jmax": 4}}
+                "jmax"
+                "amax"
+                "vmax"
```

Comment:
Minor nit, but if you *know* that it's always jmax, amax, vmax, then it would be clearer to have those be individual arguments to this method. On the other hand if this is just an example, then your API is great, and it'd be helpful to note that this is merely an example.

Reply:
For now it's only those parameters but there will be (possibly many) more in the future so I'll keep it like this. Also, the other systems will very likely get config params as well.